applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -527,12 +528,15 @@ def _CreateDistanceReinitializationProcess(self):
         elif (self._reinitialization_type == "parallel"):
             #TODO: move all this to solver settings
             layers = self.settings["parallel_redistance_max_layers"].GetInt()
+            interface_preserving = self.settings["parallel_redistance_preserve_interface"].GetBool()
             parallel_distance_settings = KratosMultiphysics.Parameters("""{
-                "max_levels" : 25,
-                "max_distance" : 1.0,
+                "max_levels" : 100,
+                "max_distance" : 1.0e2,
+                "preserve_interface_strictly" : false,
```

Comment:
```suggestion
                "preserve_interface" : false,
```
I think this is clear enough.